schism/compiler.ss

```diff
@@ -1612,8 +1614,7 @@
     (if (null? names)
         '()
         (cons (cons (number->leb-u8-list index)
-                    (encode-string (or (symbol->string (car names))
-                                       "<anonymous>")))
+                    (encode-string (symbol->string (car names))))
```

Comment:
Thanks! This will make debugging an issue on one of my in-progress PRs easier to debug. For some reason, one of the anonymous functions had some invalid Wasm bytes, but it's tricky to know for sure which one.